### DISCUSSION TITLE
Variation Selectors for script styles: Roundhead vs. Chancery

### DISCUSSION THREAD

**Discussion by DanielMGessel** [kovidgoyal/kitty#5435] [Q&A] [answered]
I understand Unicode variation selectors can choose between mathematical script capitals in Roundhead or Chancery style. The JuliaMono font has glyphs for both, but I am unable to select Roundhand style with a VS (although it does work by enabling JuliaMono's ss15 feature). I am unsure if I have a config issue, or if I shouldn't expect to be able to change the style with a variation selector.

**Comment by kovidgoyal** [comment, marked as answer]
This is done by harfbuzz, IIRC. Do the VS1 and VS2 selectors (i think those are the relevant selectors, off the top of my head) survive roundtripping in kitty? i.e. if you output some text containing them and then copy/paste that text are the selectors still present? If so, then kitty is passing them on to harfbuzz and you will need it pursue it there. If not let me know and I will take a look.

**Reply by DanielMGessel** [reply]
I did hexdump some cut and paste experiments, and concluded VS1 and VS2 selectors are maintained by kitty. I'll double check and then seek out harfbuzz maintainers for comment. Thanks!

**Comment by kovidgoyal** [comment]
Glyph selection/shaping is done by harfbuzz. I guess harfbuzz does not
support these. As long as the variation selectors are present when doing
copy paste from kitty, they will also be passed to harfbuzz. If they are
not present,then let me know and I will investigate.
